src/LuigisBoxBundle/ValueObject/SearchUrlBuilder.php

```diff
@@ -160,10 +101,7 @@ public function setFilters(array $filters): self
         return $this;
     }
 
-    /**
-     * @param bool|int|string $value
-     */
-    public function addMustFilter(string $key, $value): self
+    public function addMustFilter(string $key, bool|int|string $value): self
     {
```

Comment:
below conditions can be removed

Reply:
Then it will throw different exception, is it ok?